src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultTagHelperResolver.cs

```diff
@@ -19,46 +21,82 @@ public DefaultTagHelperResolver(bool designTime)
         public override IReadOnlyList<TagHelperDescriptor> GetTagHelpers(Compilation compilation)
         {
             var results = new List<TagHelperDescriptor>();
+            var errors = new ErrorSink();
```

Comment:
We don't flow this error sink out today. It'd probably be valuable to do that to inform tooling on bits that go wrong.